server/router/dispatcher.ts

```diff
@@ -6,12 +6,15 @@ import { Dispatcher } from '../models/dispatcher';
 const router = express.Router();
 const tableName = 'Dispatchers';
 
+// Get all dispatchers
+router.get('/', (req, res) => db.getAll(res, Dispatcher, tableName));
+
 // Put a driver in Dispatchers table
 router.post('/', (req, res) => {
   const { body } = req;
   const dispatcher = new Dispatcher({
+    id: uuid(),
```

Comment:
Make sure that this branch is up to date because this id should not be above the ...body line

Reply:
I think it is up to date but somehow id is still above ...body